meetings/2019-11/nov-18.md

```diff
@@ -0,0 +1,34 @@
+# Meeting Notes - 2019-11-18
+
+## Attendees
+
+- [Alan Dong](https://github.com/adong)
+- [Keisuke Kumada](https://github.com/kumada626)
+
+## Agenda
+
+From VMG:
+
+- [Force Push](https://github.com/screwdriver-cd/screwdriver/issues/1675) update during [freezeWindow](https://docs.screwdriver.cd/user-guide/configuration/workflow.html) time
+- [Disk Optimzation](https://github.com/screwdriver-cd/screwdriver/issues/1830)
+- [Parallel Fork](https://github.com/screwdriver-cd/screwdriver/issues/1710)
```

Comment:
don't see mention of parallel fork in the discussion notes; did we not get to that part in the meeting?

Reply:
I recalled, it was not discussed.